modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/util/IgniteMath.java

```diff
@@ -270,6 +272,15 @@ public static byte divideExact(byte x, byte y) {
         return (byte) (x / y);
     }
 
+    /** Decimal division. */
```

Comment:
the is similar `org.apache.ignite.internal.sql.engine.exec.exp.IgniteSqlFunctions#decimalDivide`. Does it make sense to leave only one of them?

Reply:
Fixed.

Reply:
let's move old method to IgniteMath to have all arithmetic related method in a single place